src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.cs

```diff
@@ -1168,9 +1168,14 @@ public static CultureInfo GetCultureInfo(string name, bool predefinedOnly)
 
             if (predefinedOnly && !GlobalizationMode.Invariant)
             {
-                return GlobalizationMode.UseNls ?
-                    NlsGetPredefinedCultureInfo(name) :
-                    IcuGetPredefinedCultureInfo(name);
+                if (GlobalizationMode.UseNls ? !CultureData.NlsIsEnsurePredefinedLocaleName(name): !CultureData.IcuIsEnsurePredefinedLocaleName(name))
+                {
+                    throw new CultureNotFoundException(nameof(name), name, SR.Format(SR.Argument_InvalidPredefinedCultureName, name));
+                }
+                else
+                {
+                    return GlobalizationMode.UseNls ? NlsGetPredefinedCultureInfo(name) : IcuGetPredefinedCultureInfo(name);
+                }
```

Comment:
last comment for me :-) 

You don't need this else block. Thanks!

Reply:
Oops! got it. Thank you